src/mbed_os_tools/detect/platform_database.py

```diff
@@ -192,6 +192,7 @@
         u"1018": u"SSCI824",
         u"1019": u"TY51822R3",
         u"1022": u"RO359B",
+        u"1026": u"MAKERDIARY_NRF52840_MDK",
```

Comment:
Please reserve this number with Mbed Platform Enablement first, contact Marcelo.Salazar@arm.com for assistance.